package.json

```diff
@@ -21,5 +21,6 @@
   "prettier": {
     "printWidth": 120,
     "singleQuote": true
-  }
+  },
+  "dependencies": {}
 }
```

Comment:
Undo changes to this file, since there are none, so it doesn't show up as having been modified in this PR